src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const galleryList = document.querySelector('.gallery__list');
+const mainImg = document.querySelector('.gallery__large-img');
+
+galleryList.addEventListener('click', (event) => {
+  const item = event.target;
+
+  if (
+    item.classList.contains('gallery__thumb')
+    || item.classList.contains('list-item__link')
+  ) {
```

Comment:
```suggestion
  if (item.closest('list-item__link')) {
    ...
```
Не уверен что это сработает:)  Но можно попробовать